lib/passport-configurator.js

```diff
@@ -250,9 +250,10 @@ PassportConfigurator.prototype.configureProvider = function(name, options) {
             }
           };
           self.userModel.findOne(query, function(err, user) {
-            if (err) {
+            if (err)
```

Comment:
Nitpick: I prefer to keep `{...}` unless it's a single line, such as `if (err) return done(err);`.